src/e2e-test/java/io.cdap.plugin/actions/ReplicationActions.java

```diff
@@ -1,157 +0,0 @@
-/*
- * Copyright (c) 2023.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package io.cdap.plugin.actions;
```

Comment:
Are we shifting these actions to framework or making them common ?

Reply:
Need some more clarification on this....This already exists so do I need to let file be as it is and use the steps from here?